bin/bootstrax

```diff
@@ -200,12 +200,18 @@ dead_state = 'dead_bootstrax'
 
 # The maximum time difference (s) allowed between the timestamps in the data and the
 # duration of the run (from the runs metadeta). Fail if the difference is larger than:
-max_timetamp_diff = 2
+max_timetamp_diff = 5
```

Comment:
Do we have sufficient confidence that we know the edge cases where this large difference is necessary and handle them individually? One is noise data, and one would involve busy/HEV at the very end of a run.

Alternately, do we leave this generally large and assume that the analysts will find the issues?

Reply:
> Do we have sufficient confidence that we know the edge cases where this large difference is necessary and handle them individually? One is noise data, and one would involve busy/HEV at the very end of a run.

I'm not entirely sure. You probably know better than I do. 

> Alternately, do we leave this generally large and assume that the analysts will find the issues?
This is indeed the approach I'm adopting here. You can consider 5 seconds reasonably large.